query-engine/connector-test-kit/src/test/scala/writes/topLevelMutations/DeleteManyMutationRelationsSpec.scala

```diff
@@ -107,58 +107,6 @@ class DeleteManyMutationRelationsSpec extends FlatSpec with Matchers with ApiSpe
 
   }
 
-  "a P1! to C1! relation " should "error when deleting the parent" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Can go.